lib/pure/os.nim

```diff
@@ -1458,16 +1466,13 @@ proc normalizePath*(path: var string) {.rtl, extern: "nos$1", tags: [].} =
 
 proc normalizePathAux(path: var string) = normalizePath(path)
 
-proc normalizedPath*(path: string): string {.rtl, extern: "nos$1", tags: [].} =
-  ## Returns a normalized path for the current OS.
-  ##
-  ## See also:
-  ## * `absolutePath proc <#absolutePath,string>`_
-  ## * `normalizePath proc <#normalizePath,string>`_ for the in-place version
-  runnableExamples:
-    when defined(posix):
-      assert normalizedPath("a///b//..//c///d") == "a/c/d"
```

Comment:
Why remove this nice description? Restore it.

Reply:
I removed it because I would've then had to explain what `isExe` is and the only difference between `normalizedPath` and `normalizePath` is that `normalizedPath` is the outplace version so there's no point in duplicating the docs or runnableExamples or related procs, which would either be incomplete or redundant. So I'm instead simply linking to the other proc, which has a full description.

Duplicating docs has the same downsides as duplicating code, and whatever's omitted / slightly different always leads to questioning whether there's some actual subtle difference, which is not the case here. it's just inplace vs outplace.